Scripts/Macros.cmd.txt

```diff
@@ -0,0 +1,7 @@
+b=%VFS_SCRIPTSDIR%\BuildGVFSForWindows.bat $*
+br=%VFS_SCRIPTSDIR%\BuildGVFSForWindows.bat Release $*
+ut=%VFS_SCRIPTSDIR%\RunUnitTests.bat $*
+utr=%VFS_SCRIPTSDIR%\RunUnitTests.bat Release $*
+ft=%VFS_SCRIPTSDIR%\RunFunctionalTests.bat $*

```

Comment:
Nit: explicitly pass in Debug here, so that I can pass in extra params and not have to remember to specify that param. Same for all of the macros, be explicit about the Debug param.

Reply:
Great point, thanks.